pkg/utils/utils.go

```diff
@@ -111,6 +111,13 @@ func (b Bytes) MacAddressWide() (out string, err error) {
 	return Bytes(bts).MacAddress()
 }
 
+// SwapCdabFloat32 swaps bytes from ABCD to CDBA, then converts to float32.
```

Comment:
should the `CDBA` in the comment be `CDAB`?

Reply:
Yes it should. Thanks.